scripts/cargo-install-all.sh

```diff
@@ -109,12 +109,14 @@ done
 (
   set -x
   # shellcheck disable=SC2086 # Don't want to double quote $rust_version
-  $cargo $maybeRustVersion build $maybeReleaseFlag "${binArgs[@]}"
+  $cargo $maybeRustVersion build $maybeReleaseFlag --bins
 )
 
 mkdir -p "$installDir/bin"
-for bin in "${BINS[@]}"; do
-  cp -fv "target/$buildVariant/$bin" "$installDir"/bin
+for bin in "target/$buildVariant/"*; do
```

Comment:
same as https://github.com/solana-labs/solana/pull/12337/files#r493815963

Reply:
Idea here was basically to get rid of the manually specified bin list, so we don't have to maintain it